src/sentry/models/release.py

```diff
@@ -622,6 +622,7 @@ def set_commits(self, commit_list):
                 )
                 group = Group.objects.get(id=group_id)
                 group.update(status=GroupStatus.RESOLVED)
+                remove_group_from_inbox(group)
                 metrics.incr("group.resolved", instance="in_commit", skip_internal=True)
 
             issue_resolved.send_robust(
```

Comment:
Interesting that this signal isn't triggered in the other places where groups are resolved :thinking:

Reply:
Yeah I feel like we'd want it to...to send the webhooks and record the analytics. Maybe we can look into whether this is causing a bug somewhere.